### DISCUSSION TITLE
Help if this is possible at all

### DISCUSSION THREAD

**Discussion by beginner984** [satijalab/seurat#4522] [Q&A] [answered]
Hello and thanks in advance

I want to compare the `immune cell complement `identified in the ascites fluid of `mouse model` using `CYTOF` with the immune cell complement identified in the ascites fluid from patients using `single cell RNA seq`. I came across this Nature medicine paper for getting `single cell RNA seq` where CYTOF is ours

A single-cell landscape of high-grade serous ovarian cancer

Is this possible at all with Seurat please?

If you have a clue, which Seurat tutorial guides me there please?

**Comment by timoast** [comment, marked as answer]
It sounds like you have a single-cell protein dataset (Cytof) and want to integrate it with a scRNA-seq dataset. You can try using the data integration tools in Seurat: https://satijalab.org/seurat/articles/integration_introduction.html

I would suggest trying `reduction="cca"` since you're integrating different data modalities.

Please try to use informative discussion topic titles, "Help if this is possible at all" does not give any indication as to what this discussion topic is about and does not enable others who might have the same question to find information.

**Comment by fi1d18** [comment]
Sorry 

I tried this but the procedure never finishes even never starts maybe because in CYTOF I have 200000 cells

Do you know any solution please?

<img width="1440" alt="Screenshot 2021-06-24 at 21 12 34" src="https://user-images.githubusercontent.com/86356901/123326385-13f6fe80-d531-11eb-9438-1482c48b9700.png">
